cmd/grafana-migrate/main.go

```diff
@@ -62,6 +62,11 @@ func main() {
 		log.Fatalf("❌ %v - failed to perform additional sanitizing of the dump file.", err)
 	}
 	log.Infoln("✅ migration_log statements removed")
+	// Fix char conversion (char -> chr)
+	if err := sqlite.CustomSanitize(dumpPath, `char\(10\)\)`, []byte("chr(10))")); err != nil {
```

Comment:
Does the regex & replacement need to have the double parentheses? (`char\(10\)\)` instead of `char\(10\)`).